cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml

```diff
@@ -58,27 +58,50 @@ phases:
             - |
               set -v
 
+              DESCRIPTION="AWS ParallelCluster AMI for {{ test.OperatingSystemName.outputs.stdout }}."
+              append_description () {
+                VALUE="$1"
+                if [[ ! "${VALUE}" =~ NOT_INSTALLED ]] && [[ -n "${VALUE}" ]]; then
+                  echo "Appending ${VALUE} to decription"
+                  DESCRIPTION="${DESCRIPTION} ${VALUE}"
```

Comment:
a comma may be clearer than space to separate the versions

Reply:
done